ibis/config_init.py

```diff
@@ -66,3 +66,14 @@
 
 with cf.config_prefix('bigquery'):
     cf.register_option('partition_col', 'PARTITIONTIME')
+
+
+with cf.config_prefix('pandas'):
+    cf.register_option(
+        'enable_trace',
+        False,
+        """
+Whether enable tracing for pandas execution. See ibis.panads.trace for details.
```

Comment:
sp? here

Reply:
Sorry I don't quite get that. What do you by `sp?`